code/game/objects/items/shuttle_creator.dm

```diff
@@ -0,0 +1,253 @@
+#define SHUTTLE_CREATOR_MAX_SIZE 200
+#define CUSTOM_SHUTTLE_LIMIT 4
+
+/obj/item/shuttle_creator
+	name = "Rapid Shuttle Designator"
+	icon = 'icons/obj/tools.dmi'
+	icon_state = "rsd"
+	lefthand_file = 'icons/mob/inhands/equipment/tools_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/tools_righthand.dmi'
+	desc = "A device used to define the area required for custom ships. Uses bluespace crystals to create bluespace-capable ships."
+	density = FALSE
+	anchored = FALSE
+	flags_1 = CONDUCT_1
+	item_flags = NOBLUDGEON
+	force = 0
+	throwforce = 8
+	throw_speed = 3
+	throw_range = 5
+	w_class = WEIGHT_CLASS_NORMAL
+	req_access_txt = "11"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 50)
+	resistance_flags = FIRE_PROOF
+	var/static/customShuttles = 0
```

Comment:
Why is this var static?

Reply:
To count total number of shuttles created

Reply:
That should be a global var instead.

Reply:
Alright, will make it global